src/tools/linkchecker/main.rs

```diff
@@ -23,6 +23,31 @@ use std::rc::Rc;
 
 use crate::Redirect::*;
 
+// Add linkcheck exceptions here
+// If at all possible you should use intra-doc links to avoid linkcheck issues. These
+// are cases where that does not work
+const LINKCHECK_EXCEPTIONS: &[(&str, &[&str])] = &[
+    // These are methods on slice, and `Self` does not work on primitive impls
+    // in intra-doc links (intra-doc links are weird)
+    // https://github.com/rust-lang/rust/issues/62834 is necessary to be
```

Comment:
I think you meant the issue on primitive types.
```suggestion
    // https://github.com/rust-lang/rust/issues/63351 is necessary to be
```

Reply:
oh, in my mind the generic issue is also what blocks slice, but your thing works